tests/test_fio_workload.py

```diff
@@ -16,12 +17,45 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture()
+def pod_cleanup(request):
+    self = request.node.cls
+
+    def finalizer():
+        if 'pod_obj' in self.__dict__:
```

Comment:
I believe you can use hasattr()

```python
        if hasattr(class_instance, 'pod_obj'):
            class_instance.pod_obj.delete()
```

Anyway, approving the PR and in case this is ok not to change it by you @petr-balogh @shylesh, please merge

Reply:
Done